contrib/binarization/confidence_based_Sauvola_binarization/README.md

```diff
@@ -0,0 +1,43 @@
+# Binarization
+Binarization is a technique to segment foreground from the background pixels. A simple technique for binarization is thresholding of gray-level or color document scanned images.
+## At a glance
+
+This binarization technique is an improvement over Sauvola's binarization technique. In this work, we improve the existing Sauvola's binarization technique by preserving more foreground information in the binarized document-images. In order to achieve this, we introduce a confidence score for the background pixels. 
+
+### Input images
+
+<img src="./test_images/2.jpeg" width="33%"> </img>
+<img src="./test_images/10.jpeg" width="33%"> </img>
+<img src="./test_images/new1.jpg" width="33%"> </img>
+
+### Sauvola outputs
+
+<img src="./results/2_bin_old.png" width="33%"> </img>
+<img src="./results/10_bin_old.png" width="33%"> </img>
+<img src="./results/new1_bin_old.png" width="33%"> </img>
+
+### Confidence based Sauvola outputs
+
+<img src="./results/2_bin_new.png" width="33%"> </img>
+<img src="./results/10_bin_new.png" width="33%"> </img>
+<img src="./results/new1_bin_new.png" width="33%"> </img>
+
+## Reference
+
+For details refer to this [paper](./ModifiedSauvola.pdf) 
```

Comment:
Should this simply point to the paper in this folder called "ModifiedSauvola.pdf" instead?

Reply:
Here we use ./ModifiedSauvola.pdf to pick the pdf file from the current working directory only.